inc/config.class.php

```diff
@@ -665,20 +650,19 @@ function showFormAuthentication() {
    /**
     * Print the config form for slave DB
     *
-    * @return Nothing (display)
+    * @return void
    **/
    function showFormDBSlave() {
       global $DB, $CFG_GLPI, $DBslave;
 
-      if (!Config::canUpdate()) {
-         return false;
+      if (!DBConnection::isDBSlaveActive() || !Config::canUpdate()) {
```

Comment:
I do not agree with that change

Reply:
The form tab is displayed only if `!DBConnection::isDBSlaveActive()` (see https://github.com/glpi-project/glpi/blob/42477190a201a1f5e49703be1c9af3d6fe0045cb/inc/config.class.php#L2082). Adding this test was just made to prevent failure on form display as `$DBslave` will be `null` if `!DBConnection::isDBSlaveActive()`. As this form should not be displayed in this case, I can remove the test if you want.

I found where was removed the test on `$active` var -> https://github.com/glpi-project/glpi/commit/439c40a2a253a9838320c1a8677622f02b16a7a9